app/styles/app.scss

```diff
@@ -1458,3 +1458,7 @@ nav.branded-navbar {
         padding-left: 40px;
     }
 }
+
+.datepicker.dropdown-menu {
+    background-color:#FFFFFF;
+}
```

Comment:
We should set this in the scss for the validated-input component in ember-osf.

Reply:
Problem is that the bootstrap is not rendered inside the "validated-input" element. It is actually rendered in the body of the page so the css for the validated-input component does not work on it.

Reply:
You can still move it to the SCSS file for the validated-input component, just leave it generic, like you have it, instead of putting it under the component div's class.